backend/src/test/java/com/huellapositiva/api/ProposalControllerShould.java

```diff
@@ -701,4 +700,53 @@ void return_404_when_proposal_with_volunteers_not_found() throws Exception{
                 .accept(MediaType.APPLICATION_JSON))
                 .andExpect(status().isNotFound());
     }
-}
+
+    @Test
+    void cancel_a_proposal_successfully() throws Exception {
+        //GIVEN
+        String proposalId = testData.registerESALAndProposalWithInscribedVolunteers().getId();
+        testData.createCredential("revisor@huellapositiva.com", UUID.randomUUID(), DEFAULT_PASSWORD, Roles.REVISER);
+        JwtResponseDto jwtResponseDto = loginAndGetJwtTokens(mvc, "revisor@huellapositiva.com", DEFAULT_PASSWORD);
+
+        //WHEN + THEN
+        MockHttpServletResponse fetchResponse = mvc.perform(post(FETCH_PROPOSAL_URI + proposalId + "/cancel")
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + jwtResponseDto.getAccessToken())
+                .with(csrf())
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn().getResponse();
+
+        MockHttpServletResponse fetchResponseAllProposals = mvc.perform(get(FETCH_PROPOSAL_URI + "0/5/reviser")
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + jwtResponseDto.getAccessToken())
+                .with(csrf())
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn().getResponse();
+
+        ListedProposalsDto listedProposalsDto = objectMapper.readValue(fetchResponseAllProposals.getContentAsString(), ListedProposalsDto.class);
```

Comment:
I would have done it this way (WARNING: pseudocode) ;-)
```
JpaProposal jpaProposal = jpaProposalRepository.findById(proposalId);
assertThat(jpaProposal.getStatus().contains(CANCELLED.toString())).isTrue();
```
But maybe I am missing something. PR looks good to me